server/supernova/server/synth.hpp

```diff
@@ -100,6 +100,8 @@ class abstract_synth:
 
     virtual float get(slot_index_t slot_id) const = 0;
 
+    virtual bool getMappedSymbol(slot_index_t slot_id, char * str) const = 0; //returns true (and writes characters to *str) if the control at slot_id is indeed mapped, otherwise returns false
```

Comment:
could you please use a javadoc comment (/** ... */ or ///) on the line above? this matches the documentation style of supernova and also keeps text at a reasonable width